integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,164 @@
+# Run Kobiton Automation Test on Travis CI
+This guide will demonstrate how to automation test on Kobiton devices with Travis CI.
```

Comment:
I think it should be 'how to run automation test(s)' or 'how to run automation test script(s)'